TheEthicalHackerCup/ProjectSettings/EditorBuildSettings.asset

```diff
@@ -4,5 +4,14 @@
 EditorBuildSettings:
   m_ObjectHideFlags: 0
   serializedVersion: 2
-  m_Scenes: []
+  m_Scenes:
```

Comment:
Did you add this manually or does this automatically get added somehow?

Reply:
In file -> build settings I built after adding the three assets and it created a build for me